test/colony-network-staking.js

```diff
@@ -0,0 +1,774 @@
+/* globals artifacts */
+import testHelper from "../helpers/test-helper";
+import testDataGenerator from "../helpers/test-data-generator";
+
+const EtherRouter = artifacts.require("EtherRouter");
+const IColony = artifacts.require("IColony");
+const IColonyNetwork = artifacts.require("IColonyNetwork");
+const Token = artifacts.require("Token");
+const ReputationMiningCycle = artifacts.require("ReputationMiningCycle");
+
+const BN = require("bn.js");
+
+contract("ColonyNetwork", accounts => {
```

Comment:
Rename the tested contract `ColonyNetwork ` to `ColonyNetworkStaking` to differentiate.